tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventAcceptMethodContract.scala

```diff
@@ -86,6 +86,178 @@ trait LinagoraCalendarEventAcceptMethodContract {
+
+    val response = `given`
+      .body(request)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .extract
+      .body
+      .asString
+
+    assertThatJson(response)
+      .withOptions(new Options(IGNORING_ARRAY_ORDER))
+      .inPath("methodResponses[0]")
+      .isEqualTo(
+        s"""[
+           |    "CalendarEvent/accept",
+           |    {
+           |        "accountId": "$ACCOUNT_ID",
+           |        "notAccepted": [ "$missingOrganizerIcsBlobId" ]
+           |    },
+           |    "c1"
+           |]""".stripMargin)
+  }
+
+  @Test
+  def acceptAMissingAttendeeIcsShouldReturnNotAccept(): Unit = {
+    val missingAttendeeIcsBlobId: String = uploadAndGetBlobId(ClassLoader.getSystemResourceAsStream("ics/missing_attendee.ics"))
+
+    val request: String =
+      s"""{
+         |  "using": [
+         |    "urn:ietf:params:jmap:core",
+         |    "com:linagora:params:calendar:event"],
+         |  "methodCalls": [[
+         |    "CalendarEvent/accept",
+         |    {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "blobIds": [ "$missingAttendeeIcsBlobId" ]
+         |    },
+         |    "c1"]]
+         |}""".stripMargin
+
+    val response = `given`
+      .body(request)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .extract
+      .body
+      .asString
+
+    assertThatJson(response)
+      .withOptions(new Options(IGNORING_ARRAY_ORDER))
+      .inPath("methodResponses[0]")
+      .isEqualTo(
+        s"""[
+           |    "CalendarEvent/accept",
+           |    {
+           |        "accountId": "$ACCOUNT_ID",
+           |        "notAccepted": [ "$missingAttendeeIcsBlobId" ]
+           |    },
+           |    "c1"
+           |]""".stripMargin)
+  }
+
+  @Test
+  def acceptAMissingVEventIcsShouldReturnNotAccept(): Unit = {
+    val missingVEventIcsBlobId: String = uploadAndGetBlobId(ClassLoader.getSystemResourceAsStream("ics/missing_vevent.ics"))
+
+    val request: String =
+      s"""{
+         |  "using": [
+         |    "urn:ietf:params:jmap:core",
+         |    "com:linagora:params:calendar:event"],
+         |  "methodCalls": [[
+         |    "CalendarEvent/accept",
+         |    {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "blobIds": [ "$missingVEventIcsBlobId" ]
+         |    },
+         |    "c1"]]
+         |}""".stripMargin
+
+    val response = `given`
+      .body(request)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .extract
+      .body
+      .asString
+
+    assertThatJson(response)
+      .withOptions(new Options(IGNORING_ARRAY_ORDER))
+      .inPath("methodResponses[0]")
+      .isEqualTo(
+        s"""[
+           |    "CalendarEvent/accept",
+           |    {
+           |        "accountId": "$ACCOUNT_ID",
+           |        "notAccepted": [ "$missingVEventIcsBlobId" ]
```

Comment:
Wrong format here.

We shall have details of the problems in there...

![image](https://github.com/linagora/tmail-backend/assets/6928740/4892c6ec-e4d9-48ed-a5a2-fb94efa948f0)

Please update the code, tests, documentation in order to match the syntax of `Email/set notUpdated`....